cg-spec/editors_draft.html

```diff
@@ -2542,6 +2557,16 @@ <h2>RDF-star Semantics</h2>
       and <dfn data-cite="RDF11-MT#dfn-equivalent" data-lt="equivalent">equivalence</dfn>.
     </p>
 
+    <section id="motivation-and-caveats">
```

Comment:
Leaving aside, as much as I can, my personal opinion about the content of this section, I am under the impression that 1) its purpose is redundant with §6.5.4 and §6.5.5 (formerly §6.4.4 and §6.4.5), and 2) commenting in details on things that have not been presented yet.

As such, I would consider it as a proposed replacement for those section rather than an additional introductory section.

Reply:
The important issue is that these topics are introduced early, not at the very end off the document. When authoring this PR I did overlook that they would wander upwards with the Semantics section if that section is moved upwards (as we seem to have a rough agreement on, see above).
However I still think that an overview section in the beginning of the semantics section should give a less technical introduction to the specifics of the proposed semantics.
There is most probaby some duplication that can be canceled but I would neither move the examples to the front nor all the discussion to the back of the Semantics section. 
tl;dr I'll look into this with a fresh mind soon. What I wrote is most probaby too long. That's almost always the case with my first attempts ;-)